### PR TITLE
chore (docs): auth state management doc changes

### PR DESCRIPTION
**Pull request by kevin-dp**

### CODE REVIEW

**Comment by alco** [comment]
@kevin-dp The other places on the website that have code snippets with `auth: { token: ... }` and missing `electric.connect()` calls:

  - [ ] https://electric-sql.com/docs/quickstart#authenticate and the "Instantiate" section below it.
  - [ ] https://electric-sql.com/docs/usage/data-access/client#instantiating-the-client
  - [ ] https://electric-sql.com/docs/deployment/concepts#4-app---electric
  - [ ] https://electric-sql.com/docs/deployment/supabase#authenticating-the-electric-client-connection
  - [ ] https://electric-sql.com/docs/integrations/frontend/react#electricprovider
  - [ ] Also noting here that the rest of the Frontend integrations are maintained by the community and they may need to be updated to use the new authentication/connection approach

**Comment by kevin-dp** [comment]
Hi @alco, i updated all occurences you mentioned + did a search across all files for these tokens and found a bunch of other places where they are used (mostly in the examples of each driver). All occurrences should be fixed now.

**Review by alco — APPROVED** [review]

File: docs/quickstart/index.md [hunk 1]
```diff
@@ -74,36 +74,35 @@ ELECTRIC GRANT ALL
...
 }
+const conn = await ElectricDatabase.init('my.db', '')
+const electric = await electrify(conn, schema, config)
 ```
 
-### Instantiate
+### Connect and authenticate
 
-Wrap your [SQLite driver](../integrations/drivers/index.md) with a type-safe, schema-aware [database Client](../usage/data-access/client.md):
+Connect to Electric and [authenticate](../usage/auth/index.md) the local app with the replication protocol using a JSON Web Token:
```
Comment (thruflo) [inline docs/quickstart/index.md:94]: Possibly “… and authenticate the connection”?

**Review by thruflo — APPROVED** [review]
Looks great 👍

**Pull request merged**